.github/pull_request_template.md

```diff
@@ -0,0 +1,37 @@
+## Summary
+<!--
+    Provide a link to the ticket or document which prompted this change,
+    Describe the rationale and design decisions.
+-->
+
+## Feature Flag
+<!-- If this is specific to a feature flag, which one? -->
+
+## Product Description
+<!-- For non-invisible changes, describe user-facing effects. -->
+
+## Safety Assurance
+
+- [ ] If the PR is high rish, "High Risk" label is set
+- [ ] I have confidence that this PR will not introduce a regression for the reasons below
+- [ ] Do we need to enhance manual QA test coverage ? If yes, "QA Note" label is set correctly
```

Comment:
This is really good. Not sure if `If yes, "QA Note" label is set correctly` is just about setting a label or putting in more details about what exactly need to be QAed. The later feels more useful to me.

Reply:
I basically wanted to track what all PRs needs to be flagged to QA team before every release so that we don't miss out on those. I think QA is generally quite good with processing prodcut note, wikis and determine what to test. I think if we have a good product description, that eliminates need for explicity defining what needs to be tested.